cc/tf_utils.cc

```diff
@@ -117,7 +126,9 @@ std::vector<tensorflow::Example> MakeExamples(
     game.GetPositionHistory(i, kMaxPositionHistory, &input.position_history);
 
     feature_desc.set_bytes({&input}, &features);
-    examples.push_back(MakeTfExample(features, move->search_pi.value(), game.result()));
+    examples.push_back(MakeTfExample(
+          features, move->search_pi.value(), move->Q, move->N, move->c,
```

Comment:
we're probably gonna need to change our BT code too at some point before v18, right

Reply:
this will work as-is but we'll have to change the BT code if we want to store Q & N in separate columns